Patches/Moyo light in the abyss/ThingDefs_Misc/Moyo_Apparel.xml

```diff
@@ -0,0 +1,580 @@
+				<value>
+					<ArmorRating_Sharp>6</ArmorRating_Sharp>
+					<ArmorRating_Electric>0.08</ArmorRating_Electric>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_SniperMask"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>3.5</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_SniperMask"]/apparel/layers/li[.="EyeCover"]</xpath>
+				<value>
+					<li>StrappedHead</li>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_SniperMask"]/equippedStatOffsets/ShootingAccuracyPawn</xpath>
+				<value>
+					<ShootingAccuracyPawn>2.5</ShootingAccuracyPawn>
+					<AimingAccuracy>2.5</AimingAccuracy>
+				</value>
+			</li>
+			
+			<!--Gas Mask-->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_GasMask"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_GasMask"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>1</ArmorRating_Sharp>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_GasMask"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>1.5</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_GasMask"]/apparel/layers/li[.="Overhead"]</xpath>
+				<value>
+					<li>StrappedHead</li>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_GasMask"]/equippedStatOffsets</xpath>
+				<value>
+					<SmokeSensitivity>-1</SmokeSensitivity>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_GasMask"]/apparel/bodyPartGroups</xpath>
+				<value>
+					<bodyPartGroups>
+						<li>Teeth</li>
+					</bodyPartGroups>
+				</value>
+			</li>
+			
+			<!--Gas Mask-->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalHeadSet"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalHeadSet"]/apparel/layers/li[.="Overhead"]</xpath>
+				<value>
+					<li>OnHead</li>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalHeadSet"]/apparel/bodyPartGroups</xpath>
+				<value>
+					<bodyPartGroups>
+						<li>Teeth</li>
+					</bodyPartGroups>
+				</value>
+			</li>
+			
+			<!--Diving Gear-->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_AbyssArmour"]/statBases</xpath>

```

Comment:
This looks to be power armor equivalent sort of stuff, so it may be appropriate to add partial coverage similar to standard vanilla power armor. Should be able to mostly just copy-paste the operation over and tweak it.

Reply:
Added.